modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-customizing-upgrades-for-kernel-modules_{context}"]
+= Customizing upgrades for kernel modules
+
+Use this procedure to upgrade the kernel module while running maintenance operations on the node,
+including rebooting the node, if needed. To minimize the impact on the workload running in the cluster, run the upgrade process
+sequentially, one node at a time. Because it requires knowledge of the workload utilizing the kernel module, this process is
+managed by the cluster administrator.
+
+.Prerequisites
+
+* To successfully upgrade a kernel module to a new version, the currently loaded kernel module
```

Comment:
I think you can edit these prereqs for brevity. what about: 

```suggestion
* Terminate all user application workloads on the node or move them to another node.
* Unload the currently loaded kernel module. 
```

Also, are these actually prereqs, or should they be part of the procedure to upgrade the kernel on the node? If they are, I think you should provide concrete steps and commands to do these things.

Reply:
Made these changes.